services/web/server/src/simcore_service_webserver/login/utils.py

```diff
@@ -138,12 +140,14 @@ async def send_mail(app: web.Application, msg: MIMEText):
             await smtp.starttls(validate_certs=False)
         if cfg.SMTP_USERNAME:
             log.info("Login email server ...")
-            await smtp.login(cfg.SMTP_USERNAME, cfg.SMTP_PASSWORD)
+            await smtp.login(cfg.SMTP_USERNAME, cfg.SMTP_PASSWORD.get_secret_value())
         await smtp.send_message(msg)
         await smtp.quit()
     else:
         async with aiosmtplib.SMTP(**smtp_args) as smtp:
             if cfg.SMTP_USERNAME:
                 log.info("Login email server ...")
-                await smtp.login(cfg.SMTP_USERNAME, cfg.SMTP_PASSWORD)
+                await smtp.login(
+                    cfg.SMTP_USERNAME, cfg.SMTP_PASSWORD.get_secret_value()
```

Comment:
good catch ! 

Just to be sure, can you please also double check  in the entire repo whether there is a similar case in which we doe not "reveal" the  ``PASSWORD`` ??

Reply:
From what I see this was the only place where it is used.